js/create-bookmarks-folder-from-window.js

```diff
@@ -0,0 +1,19 @@
+import { getAllTabsInCurrentWindow } from "./utils.js";
+
+async function createBookmarkFromWindow() {
+  const allTabsInWindow = await getAllTabsInCurrentWindow();
+
+  // Create a bookmarks folder with the tabs
+  chrome.bookmarks.create({ 'title': 'My Tabs Folder', 'index': 0 }, function (newFolder) {
```

Comment:
To make the bookmarks folder show up in the bookmarks bar, instead of doing 'index': 0, do 'parentId': '1'.